packages/cli/src/lib/config.ts

```diff
@@ -38,15 +42,30 @@ async function createConfig(): Promise<Config> {
   return new Config(result.jdkPath, result.androidSdkPath);
 }
 
-export async function loadOrCreateConfig(path = DEFAULT_CONFIG_PATH): Promise<Config> {
-  if (!existsSync(path)) {
-    // no new named config file found
-    if (existsSync(join(homedir(), '/.llama-pack/llama-pack-config.json'))) {
-      // old named config file found - rename it and it's folder
-      promises.rename((join(homedir(), '/.llama-pack')), (join(homedir(), '/.bubblewrap-config')));
-      promises.rename((join(homedir(), '/.bubblewrap-config/llama-pack-config.json')), path);
-    }
+async function renameConfigIfNeeded(): Promise<void> {
+  if (existsSync(join(DEFAULT_CONFIG_FOLDER, DEFAULT_CONFIG_NAME))) return;
+  // no new named config file found
+  if (!existsSync(join(LEGACY_CONFIG_FOLDER, LEGACY_CONFIG_NAME))) return;
+  // old named config file found - rename it and its folder
+  console.log('An old named config file was found, changing it now');
+  const files = await fsPromises.readdir(LEGACY_CONFIG_FOLDER);
+  const numOfFiles = files.length;
+  if (numOfFiles != 1) {
```

Comment:
nit: Can we add this comment to the numOfFiles check:

```suggestion
  // At this point, we know that's at least one file in the folder, `LEGACY_CONFIG_NAME`, so `numOfFiles`
  // will be at least `1`. We avoid destroying / moving other files in this folder.
  if (numOfFiles != 1) {
```